dev/ci/README-developers.md

```diff
@@ -179,6 +179,8 @@ but if you wish to save more time you can skip the job by setting
 
 This means you will need to change its value when the Docker image
 needs to be updated. You can do so for a single pipeline by starting
-it through the web interface.
+it through the web interface. Here is a direct link that you can use
+to trigger such a build:
+`https://gitlab.com/coq/coq/pipelines/new?var[SKIP_DOCKER]=false&ref=pr-XXXXX`
```

Comment:
Maybe add a note that you'll get a 404 page if unauthenticated.

Reply:
Good point, will do.